src/utils/array-utils.js

```diff
@@ -106,3 +106,17 @@ export function range({ from, to, step }) {
 
   return [...Array(Math.floor((to - from) / actualStep) + 1)].map((_, i) => from + (i * actualStep));
 }
+
+export function unique(items, keyFunc = item => item) {
```

Comment:
Tiny tiny thing, `ensureIsUnique` would match better the other helpers 😋

Reply:
True, good idea. Renamed!